docs/user/BYOnetwork.md

```diff
@@ -0,0 +1,23 @@
+## Supported scenarios and requirements for using existing AWS network resources
+
+The table below shows the scenarios supported with using existing/bring your own(BYO) network resources:
+
+| <div style="width:70px">Scenario</div> | <div style="width:70px">BYO resource</div> | <div style="width:500px">Requirements</div> | <div style="width:100px">Resources that will be created</dvi>|
+| :--- | :--- | :--- | :--- |
+| 1 | vpc_id | <ul><li>VPC block size must be IPv4 CIDR block with '/16' netmask (supports 65,536 IP addresses)</li><li>`DNS hostnames` and `DNS resolution` are enabled</li><li>`vpc_id` must be provided</li><li>`subnets` CIDR blocks must match with VPC CIDR IPv4 CIDR block</li></ul> | Subnets and Security Group|
```

Comment:
Change from vpc_id to VPC in the BYO Resource column to be consistent with how they are documented in the other two scenarios

Reply:
All values used in AWS are ID's and not names or tags so they need to be called out as such.

Reply:
This column refers to AWS Resource(s) for readability, and doesn't call out variables that's covered in CONFIG-VARS.md

Reply:
@thpang The BYO resource column lists the AWS resource as Mano notes. The next column Requirements shows the correlating variable name associated with the resource.